### DISCUSSION TITLE
20210911 - cannot access main menu settings

### DISCUSSION THREAD

**Discussion by Sagnasty** [rern/rAudio#536] [Fixed Issues]
Hello Rern,

I applied the latest image rAudio-1-RPi2-3-20210911 and then immediately applied the 20210927 update. 

I am experiencing very similar behavior that @johnsoca2 reported in Issue #513.

Pressing the Features, Player, Networks, or System button immediately displays the flashing rAudio logo in the center of the screen and I cannot proceed further.  Pressing back browser button returns to the previous screen.

The Power, Library, Playback, and Addons menus appear to be functioning as normal.  One exception being that pressing any icon within the library page (i.e. webradio, NAS, etc.) does not do anything.

Also, I have tried this same procedure on two separate Pi_ 3B+ Rev 1.3 machines with exactly the same behavior.

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

**Comment by rern** [comment]
Let me see the output of SSH command line:
`player-data.sh | jq .`

**Reply by Sagnasty** [reply]
Hi Rern,

Today, I powered up one of the machines that was giving me an issue and the problem disappeared with **absolutely no modifications at my side**.   All menus could be accessed as normal.   See output
[rAudio SSH output.txt](https://github.com/rern/rAudio-1/files/7281394/rAudio.SSH.output.txt)

I then repeated exactly the same steps that I performed yesterday in an attempt to reproduce the issue:
1) re-flashed using the latest image rAudio-1-RPi2-3-20210911 and 
[rAudio-1-RPi2-3-20210911.txt](https://github.com/rern/rAudio-1/files/7281398/rAudio-1-RPi2-3-20210911.txt)

2) immediately applu the 20210927 update.  
[rAudio-1-RPi2-3-20210911 with 20210927 update.txt](https://github.com/rern/rAudio-1/files/7281399/rAudio-1-RPi2-3-20210911.with.20210927.update.txt)
 
Again, all menus could be accessed as normal.

I know this makes no sense.  but this is the behavior I am now seeing.

If I see this behavior again, I will gather the SSH command output that you requested above.

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

**Comment by skycamel** [comment]
Hi,
just have the same problem, happens sporadically, even power off doesn't help.
I need access because I want to make backup (library, etc ...) to install latest image 
see attached file
cheers
[output_player_data.txt](https://github.com/rern/rAudio-1/files/7905744/output_player_data.txt)

EDIT: Problem on Linux machine and Android Rune app, but is working on Windows 7 machine

**Reply by Sagnasty** [reply]
Hello @rern,

I have some additional information to share on this issue.

I recently re-imaged all my machines from rAudio-1-64bit-20220116, latest updates applied, and then manually re-configured (backups were not applied).

At this point, everything appeared to be working correctly on all four machines.


How to generate issue:
1) Turn power off/on OR reboot from the GUI (power/reboot).

2) Establish a browser connection (Windows PC / Firefox) or iPhone in my case.

3) Open the System window.

4) Noticed that the mount indicator to the NAS is Green.

5) Close the System window.

6) Click on the "+", and select albums for example.

7) Now open the System window.

8) Noticed that the mount indicator to the NAS changed to Red (note: NAS was not in a hibernation state).

9) Wait for approx 20 minutes (CPU load can varies from 1.00 to 4.00) and the mount indicator will automatically turn green, the CPU load info is zero, and the system operates as normal (until the next time power is cycled).   If you try to unmount / remount the connection, switch screens, etc. you will result in an unresponsive browser connection until the 20 minute time span has expired.

Basically, just leave the system alone for 20-30 minutes after cycling power and you will not know that there was an issue as the system will behave correctly.

NOTE: I can easily repeat this behavior on all three of my Raspberry Pi 3B+ Rev 1.3 machines but CANNOT get this to occur on the Raspberry Pi 3B Rev 1.2 machine.  All machines have almost identical configurations (with the exception of host name and IP address).

Here is a SSH output of player-data.sh | jq command while I waited for the 20 minute time to expire: 
[rAudio SSH output 1-23-2022 .txt](https://github.com/rern/rAudio-1/files/7920802/rAudio.SSH.output.1-23-2022.txt)


Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

**Reply by Sagnasty** [reply]
Updated procedure to repeat issue.

**Reply by rern** [reply]
Hi Jim,

I've followed your steps but my RPi 4 and RPi Zero didn't drop the mounted NAS.

Try:
- If it's a networking problem - Switch to wired LAN.
- If it's a file server problem - Create a new share with an album or two on another machine and add the share.
